dll/win32/msafd/misc/dllmain.c

```diff
@@ -3500,11 +3545,10 @@ SetSocketInformation(PSOCKET_INFORMATION Socket,
     if (Status == STATUS_PENDING)
     {
         TRACE("Leaving (Pending)\n");
-        return WSA_IO_PENDING;
+        // return MsafdReturnWithErrno(Status, NULL, IOSB->Information, NULL);
```

Comment:
I think you can just remove this block, the "Pending" trace adds no additional information compared to the Status/Information one.